modules/photons_control/device_finder.py

```diff
@@ -449,28 +462,43 @@ def from_options(kls, options, finder=None):
 class Point:
     """Used as the value in the InfoPoints enum"""
 
-    def __init__(self, msg, keys, refresh):
+    def __init__(self, msg, keys, refresh, condition=None):
         self.msg = msg
         self.keys = keys
         self.refresh = refresh
+        self.condition = condition
 
 
 class InfoPoints(enum.Enum):
     """
-    Enum used to determine what information is required for what keys
+    Enum used to determine which packet should be sent for each device type
+    to return the right data for the specified keys
     """
 
+    VERSION = Point(DeviceMessages.GetVersion(), ["product_id", "cap"], None)
```

Comment:
```suggestion
    VERSION = Point(DeviceMessages.GetVersion(), ["product_id", "vendor_id"], None)
```

surely

Reply:
hmm, I thought I was adding reviews rather than making the comments, ignore this one